theano/sandbox/cuda/opt.py

```diff
@@ -441,6 +441,24 @@ def local_gpu_dot_to_dot22(node):
     return False
 
 
+@local_optimizer([gpu_from_host, host_from_gpu])
```

Comment:
I think this should match all ops, since the `node` you are testing is not the transfer node itself.